db/migrate/20220224202612_add_users_ref_to_comments.rb

```diff
@@ -1,5 +1,6 @@
 class AddUsersRefToComments < ActiveRecord::Migration[7.0]
   def change
     add_reference :comments, :user, null: false, foreign_key: true
+    add_index :user
   end
```

Comment:
- [ ] While creating the comment, we need to add the relationship with the comment table and `post_id` because the comment belongs to the post.